CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- [test] Bindgen 0.71 breaks test cases when crosscompiling. Using bindgen 0.70.1 still works.
```

Comment:
I'm not so sure if it make sense to add a changelog entry for a CI fix. Usually a changelog is intented for updates in relation to the library/binary.

Reply:
I think it's fine to put this here. Sometimes this file is called NEWS, which is what this is. It's a useful thing to know and I can't think of another place more likely to be seen than here.

Is it just cargo-deb that gets broken, rather than cross-compiling in general? I've previously run into a few edge-cases where cargo-deb falls over. I normally now use it with `--no-build` and sort that out myself.

Reply:
Sounds good. It would be nice to have a reference to https://github.com/actions/runner-images/issues/10636 in the comment as we need to install bindgen manually as it was removed from the base image runner.